syntax = "proto3";

package recommendation;

// gRPC 서비스 정의
service UserRecommendationService {
  // 사용자 추천 요청 및 응답
  rpc GetRecommendedUsers(RecommendationRequest) returns (RecommendationResponse);
}

// 요청 메시지: 사용자 ID 리스트
message RecommendationRequest {
  repeated int64 userIds = 1;
  string elderlyDetail = 2;
}

// 응답 메시지: 추천된 사용자 ID 리스트
message RecommendationResponse {
  repeated int64 recommendedUserIds = 1;
}
